product_attribute_value_adj/views/product_attribute_value_views.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="product_attribute_value_form_view" model="ir.ui.view">
+        <field name="name">product.config.product.attribute.value.form.view</field>
+        <field name="model">product.attribute.value</field>
+        <field name="inherit_id" ref="product_configurator.product_attribute_value_form_view"/>
+        <field name="arch" type="xml">
+            <xpath expr="//form" position="attributes">
+                <attribute name="duplicate" version="10.0">false</attribute>
```

Comment:
`version="10.0"`

What is this about?

Reply:
Maybe I should remove this, since it doesn't have any affection.